example/App.tsx

```diff
@@ -34,51 +44,122 @@ const styles = StyleSheet.create({
         marginBottom: 5,
     },
     buttons: {
+        marginBottom: 10,
         flex: 0.5,
         justifyContent: 'space-around',
         alignItems: 'center',
     },
 });
 
+const DEFAULT_CONFIGURATION: LocationConfiguration = {
+    desiredAccuracy: 'BalancedPower' as AccuracyLevel,
```

Comment:
It would probably be better to export an enum from the package for the AccuracyLevel

Reply:
Yeah I think about it and finally I got lazy to create the enum in the package (it didn't exist)